src/scripts/librarymenu.js

```diff
@@ -925,6 +961,8 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
         updateUserInHeader();
     });
     events.on(playbackManager, "playerchange", updateCastIcon);
+    events.on(syncplayManager, "SyncplayEnabled", updateSyncplayIcon);
+    events.on(syncplayManager, "SyncplayError", updateSyncplayErrorIcon);
```

Comment:
where's paired `events.off()`?

Reply:
The whole file has no `events.off()`. I'm not sure there's a place where I can unregister those callbacks.

Reply:
This means we would need to refactor this whole thing later on...